demisto_sdk/commands/common/update_id_set.py

```diff
@@ -906,6 +926,16 @@ def process_indicator_types(file_path: str, print_logs: bool, all_integrations:
     return res
 
 
+def process_pack_metadata(file_path: str, print_logs: bool):
```

Comment:
This name is a bit unclear, not sure what kind of processing happens here, especially since this essentially is a wrapper for `get_pack_metadata_data`. I would consider adding the try-except mechanism there

Reply:
done